packages/bio-parsers/src/genbankToJson.js

```diff
@@ -747,6 +757,9 @@ function genbankToJson(string, options = {}) {
         inclusive1BasedStart,
         inclusive1BasedEnd
       );
+    } else if (feat.locations && feat.locations.length > 1) {
+      // Sort locations for linear sequences
+      feat.locations.sort((a, b) => a.start - b.start);
```

Comment:
maybe we can only reverse the order for some special case described in the ticket, right? It seems this code will work for all cases besides the special case